src/DateRangePicker/Calendar.styles.tsx

```diff
@@ -247,13 +247,13 @@ export const CalendarContainer = styled.div`
       color ${ACCENT_MEDIUM};
     }
 
-    &.is-selected:not(.is-passive) {
-      background: ${ACCENT_MEDIUM};
-      color: white;
-      border-radius: 2px;
+    &.is-selected {
+      background: ${ACCENT_MEDIUM} !important;
```

Comment:
I think `!important` should be used as last resource. is there another option ?

Reply:
agree, will try, but there is issue with specificity

Reply:
@pettomartino done